webpack.config.js

```diff
@@ -0,0 +1,9 @@
+const { merge } = require("webpack-merge");
+
+const commonConfig = require("./webpackConfigurations/webpack.common.js");
```

Comment:
using build name instead of webpackConfiguration isn't more descriptive and widely used?